backend/src/flotilla/settings/settings.py

```diff
@@ -17,6 +17,11 @@ class Settings(BaseSettings):
     REQUEST_TIMEOUT: float = Field(default=5, env="REQUEST_TIMEOUT")
     ISAR_CLIENT_ID: str = Field(default="", env="ISAR_CLIENT_ID")
     ISAR_APP_SCOPE: str = Field(default="", env="ISAR_APP_SCOPE")
+    SQLALCHEMY_DATABASE_URL: str = Field(
+        default="sqlite:///:memory:", env="SQLALCHEMY_DATABSE_URL"
+    )
+    SKIP: int = Field(default=0, env="SKIP")
+    LIMIT: int = Field(default=100, env="LIMIT")
```

Comment:
We should also have a MAX_LIMIT to prevent the user from overriding the pagination alltogether